assets/js/script.js

```diff
@@ -0,0 +1,68 @@
+$(document).ready(function () {
+  $("#show").click(function () {
+    $(this).toggleClass("fa-eye fa-eye-slash");
+    var type = $(this).hasClass("fa-eye-slash") ? "text" : "password";
+    $("#code").attr("type", type);
+  });
+
+
+  $("#name").on("change", function() {
+    var username = $(this).val();
+
+    if (validateName(username)) {
+      $("#name-error").html("");
+    } else {
+      $("#name-error").html("Only alphabets and space are allowed");
+    }
+
+    if (username.length < 10) {
+      $("#name-error").html("Please provide your full name");
+    }
+  });
+
+  function validateName(name) {
+    var pattern = /^[a-zA-Z-' ]+$/;
+
+    return $.trim(name).match(pattern) ? true : false;
+  }
+
+  $("#mail").on("change", function() {
+    var emailid = $(this).val();
+
+    if (validateEmail(emailid)) {
+      $(this).css({
+        color: "green",
+        border: "1px solid green"
+      });
+      $("#mail-error").html("");
+    } else {
```

Comment:
Follow proper coding standards, `else {` will start from new line.

Reply:
I have used the same pattern in each file.